RELEASE.txt

```diff
@@ -37,24 +53,31 @@ How to make a new release of ``skimage``
 
    git push --tags origin master
 
-- Publish on PyPi::
+  (or ``git push --tags upstream master``, depending on your workflow.)
```

Comment:
How about just `git push --tags upstream master` (where `upstream` is the name of the `github.com:scikit-image/scikit-image` repository).